src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -202,7 +204,13 @@ public async Task<BundleResponse> Handle(BundleRequest request, CancellationToke
                     };
 
                     await ExecuteAllRequests(responseBundle);
-                    return new BundleResponse(responseBundle.ToResourceElement());
+                    var response = new BundleResponse(responseBundle.ToResourceElement());
+
+                    int successfulRequestCount = responseBundle.Entry.Count(entry => entry.Response.Status.StartsWith("2", StringComparison.Ordinal));
```

Comment:
>entry.Response.Status.StartsWith("2" [](http://example.com/codeflow?start=85&length=36)

lol.

What do we return for delete request?

It's also so freaking brittle to do this... It's just a string with no restrictions, what if someone one day will come and change `200 Created` to `Created 200`?

Can we check for OperationOutcome with errors in response?

Reply:
I changed the notification to send results sorted by response code. We will check for successful codes in PaaS.